app/unit-tests/src/org/commcare/android/tests/DemoUserRestoreTest.java

```diff
@@ -41,16 +42,6 @@
     private final static String REF_BASE_DIR =
             "jr://resource/commcare-apps/demo_user_restore/";
 
-    @Test
-    public void loginUsingDemoUserWithoutRestore() {
```

Comment:
Removed this since same steps as in this test are executed in `demoUserRestoreAndUpdateTest` as well